packages/eslint-plugin/src/rules/base-to-string.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  TSESTree,
+  AST_NODE_TYPES,
+} from '@typescript-eslint/experimental-utils';
+import * as ts from 'typescript';
+
+import * as util from '../util';
+import { getTypeName } from '../util';
+
+type MessageIds = 'baseToString';
+
+enum Usefulness {
+  Always,
+  Never = 'will',
+  Sometimes = 'may',
+}
+
+export default util.createRule<[], MessageIds>({
```

Comment:
if there is no options and you are not using `MessageIds` in other places you can omit this
```suggestion
export default util.createRule({
```